Sources/OpenPass/OpenPassManager.swift

```diff
@@ -97,13 +97,28 @@ public final class OpenPassManager {
             baseURL = Self.defaultBaseURL
         }
 
-        let clientId = Bundle.main.object(forInfoDictionaryKey: "OpenPassClientId") as? String
-        let redirectHost = Bundle.main.object(forInfoDictionaryKey: "OpenPassRedirectHost") as? String
+        let clientId: String
+        if let bundleClientId = Bundle.main.object(forInfoDictionaryKey: "OpenPassClientId") as? String {
+            clientId = bundleClientId
+        } else if let settingsClientId = OpenPassSettings.shared.clientId {
+            clientId = settingsClientId
+        } else {
+            clientId = ""
+        }
+
+        let redirectHost: String
+        if let bundleRedirectHost = Bundle.main.object(forInfoDictionaryKey: "OpenPassRedirectHost") as? String {
+            redirectHost = bundleRedirectHost
+        } else if let settingsRedirectHost = OpenPassSettings.shared.redirectHost {
+            redirectHost = settingsRedirectHost
+        } else {
+            redirectHost = ""
+        }
 
         self.init(
```

Comment:
Now that we can set these programmatically, is there any value on having unit test coverage that those set are used in the sign in flow?